addons/medical_treatment/functions/fnc_scanMedicalItems.sqf

```diff
@@ -0,0 +1,24 @@
+#include "script_component.hpp"
+/*
+ * Author: Salluci
+ * Caches all item classnames used in ACE_Medical_Treatment_Actions
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * call ace_medical_treatment_fnc_getAllMedicalItems
+ *
+ * Public: No
+ */
+
+private _list = ["ACE_suture"]; // ugly workaround for sutures
+private _config = configFile >> QGVAR(actions);
+{
+    _list append (getArray (_x >> "items"));
+} forEach ("true" configClasses _config);
+
+uiNamespace setVariable [QGVAR(treatmentItems), _list arrayIntersect _list]
```

Comment:
```suggestion
uiNamespace setVariable [QGVAR(treatmentItems), compileFinal str (_list arrayIntersect _list)]
```
this protects the var

in 2.14 we can actually `compileFinal` arrays - https://community.bistudio.com/wiki/compileFinal
(although it's broken on dev branch)